_includes/selector.html

```diff
@@ -481,34 +481,54 @@
-                var index_url = `--${this.highlightFlag("extra-index-url")}=https://pypi.nvidia.com`;
                 var cuda_suffix = "-cu12";
                 var indentation = "    ";
 
                 if (this.active_pip_cuda_ver === "11.2 - 11.8") {
                     cuda_suffix = "-cu11";
                 }
 
-                var libraryToPkg = (pkg) => {
-                    pkg = pkg.toLowerCase();
-                    if (pkg === "cucim") return pkg;
-                    if (pkg == "cuspatial/cuproj") return "cuspatial" + cuda_suffix + " cuproj" + cuda_suffix;
-                    return pkg + cuda_suffix;
+                // Change index depending on stable vs nightly for pip
+                // Also add versioning commands for nightly installs to prevent --pre usage
+                // This has dupilicate code, but makes for easier edits in the future
+                if (this.active_release === "Stable") {
+                    index_url = `--${this.highlightFlag("extra-index-url")}=https://pypi.nvidia.com`;
+                    var libraryToPkg = (pkg) => {
+                        pkg = pkg.toLowerCase();
+                        if (pkg === "cucim") return pkg; // Remove this line in 23.12 when stable cucim is out on our index
+                        if (pkg == "cuspatial/cuproj") return "cuspatial" + cuda_suffix + " cuproj" + cuda_suffix;
+                        return pkg + cuda_suffix;
+                    }
+                }
+                else {
+                    index_url = `--${this.highlightFlag("extra-index-url")}=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple`;
+                    cuda_suffix = cuda_suffix + ">={{ site.data.releases.nightly.version }}.0a0\"";
+                    var libraryToPkg = (pkg) => {
+                        pkg = pkg.toLowerCase();
+                        if (pkg == "cuspatial/cuproj") return '"cuspatial' + cuda_suffix + ' "cuproj' + cuda_suffix;
+                        return "\"" + pkg + cuda_suffix;
+                    }
                 }
+
                 if (this.active_packages.length === 1 && this.active_packages[0] === "Choose Specific Packages") {
                     return "Select at least one package.";
                 } else if (this.active_packages[0] === 'Standard') {
                     var pkgs = this.additional_pip_packages.map(libraryToPkg);
                 } else {
                     var pkgs = this.active_packages
-                        .filter(pkg => pkg !== "Choose Specific Packages" && pkg !== "cuSignal")
+                        .filter(pkg => pkg !== "Choose Specific Packages")
                         .map(libraryToPkg);
                 }
 
                 if (pkgs.length === 1 && pkgs[0] === "cucim") {
                     index_url = "";
                 }
 
+                //for every 3rd package add a new line with a "\" character
+                for (var i = 3; i < pkgs.length; i += 4) {
```

Comment:
This behavior change should match the comment. Currently the second (and later) line of packages will contain 4 packages instead of 3. It gets a little too long.
```suggestion
                for (var i = 3; i < pkgs.length; i += 3) {
```